src/common/options/mds-client.yaml.in

```diff
@@ -153,7 +153,6 @@ options:
     sets how many   seconds a client waits to release capabilities that it no
     longer needs in case the capabilities are needed for another user space
     operation.
-  with_legacy: true
```

Comment:
Did you intend to remove this line?

Reply:
Yes,was giving some errors ,though not required.

Reply:
Do you know what it does? Can you tell me why it's not required? :)

Reply:
with_legacy = true means its inherited the older config options as legacy because the yaml.in method is introduced recently.

Reply:
I don't think that's quite right. Right now, our config structure is a class we can lookup keys and values from by calling a lookup function, but the older version we had contained each of these keys as a public member we could access directly. (You'll see this happening a lot in the code, something like ```g_ceph_context->_conf-> client_caps_release_delay```.) with_legacy indicates whether we need the config variable to be present in that direct-access structure.

I'm not certain if having it *off* means the variable is still present and unset, or left out — if having with_legacy set was causing build errors I'm concerned we may be leaving around detritus that is looking at an unset value with this change. Happily, running "git grep" indicates that's not the case!

Reply:
@neesingh-rh ping? were you able to get this sorted out?

Reply:
Yup,I did handle the config options in Client::handle_conf_change.

Reply:
@neesingh-rh Is removing this line really required?